documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,405 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere, {rhv-full}, or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, known issues, and resolved issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Migration from OpenStack moves to being a fully supported feature
+
+In this version of {project-short}, migration using {osp} source providers graduated from a Technology Preview feature to a fully supported feature.
+
+.Integration of the create and update provider user interface
+
+The user interface of the source providers and destination providers now aligns with the look and feel of the {ocp} web console and displays the current data.
+
+.Standalone UI
+
+The old UI of {project-short} 2.3 cannot run by setting `feature_ui: true` in ForkliftController anymore.
+
+.Improved errors logged in populator pods
+
+In earlier releases of {project-short} {project-version}, populator pods were always restarted on failure. This made gathering logs from the failed pods difficult. {project-short} 2.5.3 has a maximum number of three restarts of populator pods. On the third and final time, the populator pod remains in the fail status, and its logs can then be easily gathered by `must-gather` and by `forklift-controller`, to know this step has failed. link:https://issues.redhat.com/browse/MTV-818[(MTV-818)]
+
+.The `VirtualMachineClusterFlavor` CRDs replace {ocp-name} templates
+
+The Virtual Machine Flavor API replaces {ocp-name} templates. {project-short} currently falls back to using {ocp-name} templates when a relevant preference is not available.
+
+.Full support for migration from OVA
+
+{project-short} {project-version} supports migration from OVA.
+
+.{ocp-name} Console `Create` and `Edit` pages replace Forms and Wizards
+
+Forms and Wizards that use `@migtools/lib-ui` do not function as expected when running inside the plugin. The migtool library was built as a standalone application. The `Create` and `Edit` pages in the {ocp-name} Console replace the Forms and Wizards. link:https://issues.redhat.com/browse/MTV-615[(MTV-615)]
+
+.Scan OVA files in the root folder of an NFS share
```

Comment:
that was done in 2.5.1, see https://issues.redhat.com/browse/MTV-697

Reply:
removed